app/lib/SecvisogramPage/View/FormEditorTab/Document.js

```diff
@@ -78,23 +80,27 @@ export default React.memo(
             />
             <Distribution {...documentLevelMetaDataProps('distribution')} />
             <Lang
+              {...documentLevelMetaDataProps('lang')}
```

Comment:
Not directly related to this change request: I'm not aware of this standard and naively tried to give it a value expressing "American English". `en-US` doesn't seem to be right and the referenced https://www.iana.org/assignments/language-subtag-registry/language-subtag-registry doesn't mention `American` or `British` at all. Do I miss sth.?

Reply:
@bentolor The [BCP 47](https://www.ietf.org/rfc/bcp/bcp47.txt) is commonly used to identify languages. The languages consists of different subtags - each separated with `-`. The list of types and the rules for their combination is given in  Figure 1 of [BCP 47](https://www.ietf.org/rfc/bcp/bcp47.txt). A list of all subtags and their types is in the [registry](https://www.iana.org/assignments/language-subtag-registry/language-subtag-registry). 

`en-US` is a valid example. It has the language subtag `en` and the region subtag `US`. It identifies the "English" as used in the "United States" (cp. "Region subtags are used to indicate linguistic variations associated with or appropriate to a specific country, territory, or region.")

As identified in #53 Secvisogram handles language tags incorrectly as it:
1. doesn't split on `-`,
2. therefore doesn't allow combinations of subtags,
3. doesn't check that the combination follows the rules of the position of types

Does that answer your question?

Reply:
Thanks for you extensive explanation and clarification! 

Yes – _`en-US` is a valid example_ resolves my confusion. 

I'm no longer sure what caused the confusion: Probably a combination of a) the impression that the existing error was caused by the RegEx (rather than the incomplete validation implementation) and b) I failed to spot the `Subtag: US` entry in the registry.

Again, thanks for helping me out, here!